LayoutTest.xcodeproj/xcshareddata/xcschemes/LayoutTestBase.xcscheme

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Presumably this is just making the schema 'shared'?
Why does carthage need this?